Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -469,5 +485,46 @@ void UpdateMaxLines()
 #endif
 			}
 		}
+
+		void UpdatePadding()
+		{
+			if (Element.Padding.IsEmpty)
+				return;
+
+#if __MOBILE__
+			var formsLabel = Control as FormsLabel;
+			if (formsLabel == null)
+			{
+				Debug.WriteLine($"{nameof(LabelRenderer)}: On iOS, a Label created with no padding will ignore padding changes");
+				return;
+			}
+
+			formsLabel.TextInsets = new UIEdgeInsets(
+					(float)Element.Padding.Top,
+					(float)Element.Padding.Left,
+					(float)Element.Padding.Bottom,
+					(float)Element.Padding.Right);
+			UpdateLayout();
+#endif
+		}
+
+#if __MOBILE__
+		class FormsLabel : NativeLabel
```

Comment:
Wondering if this should be moved to its own file...

Reply:
@adrianknight89 Nah, it's an internal small class for specific purposes. There's lots of similar classes already.